src/source/Ice/IceAgent.c

```diff
@@ -256,12 +256,15 @@ STATUS iceAgentAddRemoteCandidate(PIceAgent pIceAgent, PCHAR pIceCandidateString
     PCHAR curr, tail, next;
     UINT32 tokenLen, portValue, remoteCandidateCount, i;
     BOOL foundIpAndPort = FALSE, freeIceCandidateIfFail = TRUE;
+    BOOL foundIp6 = FALSE;
+    CHAR ip6Buf[KVS_MAX_IPV6_ADDRESS_STRING_LEN+1];
```

Comment:
Might need to setup some editor auto-formatting preferences. In this case, the '+' sign should have spaces around it

Reply:
can use KVS_IP_ADDRESS_STRING_BUFFER_LEN instead

Reply:
Do we have a standard format config? Something like .clang-format?

Ah, didn't realize that we have `KVS_IP_ADDRESS_STRING_BUFFER_LEN`. I'll update it. Thanks.